device/bgp/evpn/bgpevpnethernetsegment.yaml

```diff
@@ -0,0 +1,64 @@
+components:
+  schemas:
+    Bgp.EvpnEthernetSegment:
+      type: object
+      properties:
+        esi:
+          description: >-
+            10-octet Ethernet segment identifier (esi)
+            Example - for example for multi-home nonZero ESI '10000000000000000000' .
+          type: string          
+          format: hex
+          default : '00000000000000000000'
+        active_mode:
+          description: >-
+            Single Active or All Active mode Redundancy mode selection for Multi-home.
+          type: string
+          enum:
+          - single_active
+          - all_active
+          default: all_active
+        esi_label:
+          description: >-
+            The label value to be advertised as ESI Label in ESI Label Extended Community.
+            This is included in Ethernet Auto-discovery Routes per ES advertised by a PE, 
+          type: integer
+          minimum: 0
+          maximum: 16777215
+          default: 0
+        advertise_ad_evi_route:
```

Comment:
ad  per evi should get advertised based on configuration,why do we have a knob here?.

Reply:
Removed this.